Indaba_2022_Prac_Template.ipynb

```diff
@@ -6,7 +6,7 @@
         "id": "m2s4kN_QPQVe"
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    # @title Answer to code task (Don't peek!) 
</pre><p>I wonder if we could have a slightly different message than 'Don't peek!', maybe 'Try not to peek until you've given it a good try!'. I am trying to go for something that both suggests that they should try it themselves and also that it is ok to peek if they need to. Thoughts?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deep-learning-indaba/indaba-pracs-2022/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='87'/>

Reply:
Sounds good, updated :)